src/shared/features/AppletSettings/ReportConfigSetting/ReportConfigSetting.tsx

```diff
@@ -40,9 +43,15 @@ import {
 } from 'shared/utils';
 import { reportConfig } from 'modules/Builder/state';
 
+import { useCurrentActivity, useCurrentActivityFlow } from 'modules/Builder/hooks';
```

Comment:
pls remove empty line before and add after

Reply:
fixed